src/Integration/Helpers/KnownUIContextsWrapper.cs

```diff
@@ -74,5 +84,9 @@ public class KnownUIContextsWrapper : IKnownUIContexts
                 KnownUIContexts.VBProjectContext.UIContextChanged -= value;
             }
         }
+
+        public IUIContext SolutionExistsAndFullyLoadedContext => new UIContextWrapper(KnownUIContexts.SolutionExistsAndFullyLoadedContext);
+
+        public IUIContext SolutionExistsAndNotBuildingAndNotDebuggingContext => new UIContextWrapper(KnownUIContexts.SolutionBuildingContext);
```

Comment:
I've added the contexts needed by the binding controller. The source controlled file system and error list controller will need other contexts. I'll handle those separately.